charts/logan/values.yaml

```diff
@@ -67,6 +67,12 @@ kubernetesClusterID:
 # e.g. production-cluster
 kubernetesClusterName:
 
+# -- Kubernetes Security Context privileged flag
+# By default, this is false and not required to be set in OKE cluster
+# In some special Kubernetes environments where SELinux mode is enforcing, we require this flag to be set to ture
```

Comment:
Suggest rephrasing:
"Default: 'false'. This is not a required for OKE clusters. In Kubernetes environments where SELinux mode is enforced, set this flag to 'true' to allow fluentd pods to access log files.”